views/agency-requests/edit_participant.php

```diff
@@ -0,0 +1,345 @@
+					<button type="button" class="close" data-dismiss="alert" aria-label="Close">
+						<span aria-hidden="true">&times;</span>
+					</button>
+					<strong>Participant Update Success</strong> Participant name information has been updated.
+				</div>
+
+			<?php
+			}
+		
+	}else{
+		?>
+			<div class="alert alert-danger alert-dismissible fade show" role="alert">
+			<button type="button" class="close" data-dismiss="alert" aria-label="Close">
+			<span aria-hidden="true">&times;</span>
+			</button>
+				<strong>Error: </strong> Participant name information update has failed. Only 1 character allowed for middle name initial
+			</div>
+			  <?php
+	}
+	
+	/**
+	 * Checks to see if the updated phone number already exists or 
+	 * not. If it exists, create an update statement; 
+	 * if not, create an insert statement
+	 * @params $db database object from globals
+	 * @params $formid id of the form associated with the participant 
+	**/
+	function updateInsert( $db, $formid){
+	global $db, $params;
+	$peopleid = rawurldecode(implode('/', $params));
+		$pphone = $_POST['pphone-update'];
+		$sphone = $_POST['sphone-update'];
+		$dphone = $_POST['dphone-update'];
+		$ephone = $_POST['ephone-update'];
+		$error = 0;
+		$errorArray = array();
+		$phoneNumbers = ["Primary"=>$pphone, "Secondary"=>$sphone, "Day"=>$dphone, "Evening"=>$ephone];
+		$resArray = array();
+		foreach($phoneNumbers as $key => $values){
+			$res2 = $db->query("UPDATE formphonenumbers SET phonenumber = $1 WHERE phonetype = $2 AND formid = $3 ", [$values, "$key", $formid]);
+			if($res2){
+				$res3 = $db->query("INSERT INTO formphonenumbers (formid, phonenumber, phonetype) VALUES($1, $2, $3)", [$formid, $values, "$key"]);
+				if(!$res3){
+					$error++;
+				}else{
+
+				}
+			}
+		}
+		
+		
+		if($error == 0){
+		?>
+			
+			<div class="alert alert-success alert-dismissible fade show" role="alert">
+			<button type="button" class="close" data-dismiss="alert" aria-label="Close">
+			<span aria-hidden="true">&times;</span>
+			</button>
+				<strong>Success: </strong> Participant information has updated 
+			</div>
+			<div class="">
+				<span class="cpca btn"> 
+				<i class="fa fa-arrow-left"></i><a style="text-decoration:none; color:white;"href="/ps-view-participant/<?=$peopleid?>">
+				 Back to View Participant</a>
+				</span>
+			</div>
+			
+			  <?php
+		}else{
+			
+		}
+	}
+	if(!empty($_POST['form'])){
+	updateInsert($db, $formid);
+	}
+	
+	$addressid = $_POST['addressid'];
+	$streetnumber = $_POST['house-update'];
+	$streetaddress = $_POST['street-update'];
+	$aptinfo = $_POST['apt-update'];
+	$city = $_POST['city-update'];
+	$state = $_POST['street-update'];
+	$zip = $_POST['zip-update'];
+	
+	$addressupdate = $db->query("UPDATE addresses SET addressnumber = $2, aptinfo = $3, street = $4, zipcode =$5 WHERE addressid = $1",
+								[$addressid, $streetnumber, $aptinfo, $streetaddress, $zip]);
+	if(!$addressupdate){
+		echo "it worked!";
```

Comment:
I'm not sure if you meant to leave this in or not. If this is meant to be an actual success/failure message you should create a $success Boolean and set this equal to true or false here, then display a notification accordingly.